tools/replay_gen.py

```diff
@@ -317,12 +347,21 @@ def parse_args():
                         help='Path to a directory where the replay files are generated.')
     parser.add_argument('-p', '--prefix', dest='prefix', type=str, default='',
                         help='Prefix for the replay file names.')
+    parser.add_argument(
+        '-j',
+        '--out-json',
+        dest='out_json',
+        default=False,
+        action='store_true',
+        help='Dump replay files in JSON format. By default replay files will be formatted as YAML.')
     return parser.parse_args()
 
 
 def main():
     args = parse_args()
 
+    out_json = True if not has_yaml else args.out_json
```

Comment:
Oh, I think my initial suggestion didn't work, right? Let me try again. I think the following simplifies things:

```
out_json = (not has_yaml) or args.out_json
```